pkg/util/api.go

```diff
@@ -49,13 +50,21 @@ func ToRFCFormat(timestamp time.Time) string {
 }
 
 // RequestBodyToBytes reads the request body and returns []byte with its content or an error if
-// the media type is incorrect or if the body is not a valid JSON
+// the media type is unsupported or if the body is not a valid JSON
 func RequestBodyToBytes(request *http.Request) ([]byte, error) {
 	contentType := request.Header.Get("Content-Type")
-	if !strings.Contains(contentType, "application/json") {
+	contentTypeSupported := false
```

Comment:
use `slice.StringsContaining(supportedContentTypes, contentType)`

Reply:
`slice.StringsContaining(supportedContentTypes, contentType)` actually works in reverse to what we need here. It will check that the provided request `contentType` is contained in any of the `supportedContentTypes`, but like I explained in the above comment, this will not work as the provided `contentType` will contain much more information than what we've defined in our `supportedContentTypes`. So `strings.Contains` will never match.